pyiron_base/project/generic.py

```diff
@@ -162,6 +162,11 @@ def view_mode(self):
         """
         Get viewer_mode - if viewer_mode is enable pyiron has read only access to the database.
 
+        Change it via
+        `Project('my_project').db.switch_to_viewer_mode()`
+        and
+        `Project('my_project').db.switch_to_user_mode()`
```

Comment:
```suggestion
         `Project('my_project').switch_to_viewer_mode()`
         and
         `Project('my_project').switch_to_user_mode()`
```
The database should not be accessed directly.

Reply:
Thanks I didn't know that.